entries/t/tvtropes.org.json

```diff
@@ -0,0 +1,14 @@
+{
+    "TV Tropes": {
+        "domain": "tvtropes.org",
+        "img": "tvtropes.org.png",
+        "contact": {
+            "facebook": "TVTropes",
+            "twitter": "TVTropes",
+            "email": "thestaff@tvtropes.org"
+        },
+        "categories": [
+            "entertainment"
```

Comment:
The site looks similar to Fandom which is in the category `other` along with other wikis.